MLK_Uninstall/Form1.cs

```diff
@@ -50,32 +44,43 @@ private void button1_Click(object sender, EventArgs e)
                 else
                 {
                     InstallFolderPath = Path.Combine(Environment.GetEnvironmentVariable("ProgramFiles"), "MyLittleKaraoke");
-                };
+                }
+
                 if (Directory.Exists(InstallFolderPath) == false)
                     InstallFolderPath = AppDomain.CurrentDomain.BaseDirectory;
 
-                DialogResult dialogResult = MessageBox.Show("Confirm to delete all files in " + InstallFolderPath, "Confirm uninstalling ponies? - Singing is Magic", MessageBoxButtons.OKCancel, MessageBoxIcon.Warning);
+                DialogResult dialogResult = MessageBox.Show("Confirm to delete all files in " + InstallFolderPath +".\nPlease note that this will also remove all of your installed songs."My Little Karaoke Uninstaller", MessageBoxButtons.OKCancel, MessageBoxIcon.Warning);
```

Comment:
missing `", `. Also, please change the title of the messagebox back to `Confirm uninstalling ponies?...`.